util.py

```diff
@@ -20,23 +20,53 @@
     Utility functions used in various places in the codebase.
 
 """
+from typing import List
 
-import os
 from functools import lru_cache
+from pathlib import Path
+
+
+def _find_greynir_root_folder() -> Path:
+    """Small helper function to find the root folder of Greynir."""
+    p = Path(__file__).resolve()
+    # Search for LICENSE.txt file (should be in root folder)
+    while not (p / "LICENSE.txt").is_file() and p.parent != p:
```

Comment:
I dunno, looking for LICENSE.txt feels a bit hacky, but I can't think of a good alternative method off the top of my head.

Reply:
Hmm, yeah, I couldn't think of another file that was guaranteed to be in our root and not change names.

Reply:
I guess we could change to searching for the parent folder being named "Greynir"? (I would need to rename the folder for my fork from "GreynirFork" though)

Reply:
No, just leave it as it is. The name of the repo is more likely to change in the future than LICENSE.txt.

Reply:
The most important idea for this function was that running it from anywhere within the repo should always return the same path (although I think it only gets called from the root directory anyway when the server is run normally).

Reply:
hm ok I can try using `__file__` instead :)

Reply:
`__file__` works and is cleaner (I misunderstood how the variable worked), made this much cleaner :+1:.